src/pages/search/index.page.tsx

```diff
@@ -0,0 +1,131 @@
+import { Stack } from '@/ui/Stack';
+import { Tabs } from '@/ui/Tabs';
+import { colors } from '@/ui/theme';
+import { getApplicationServerSideProps } from '@/utils/getApplicationServerSideProps';
+
+import { Scope } from '../create/OfferForm';
+
+const Search = () => {
+  const { publicRuntimeConfig } = getConfig();
+  const { t } = useTranslation();
+  const [iframeHeight, setIframeHeight] = useState(0);
+  const legacyPath = useLegacyPath();
+  const { query, ...router } = useRouter();
+  const tab = (query?.tab as Scope) ?? 'event-places';
+  const { udbMainDarkBlue } = colors;
+  const isClientSide = useIsClient();
+
+  const handleSelectTab = async (tabKey: Scope) =>
+    router.push(
+      {
+        pathname: '/search',
+        query: {
+          tab: tabKey,
+        },
+      },
+      undefined,
+      { shallow: true },
+    );
+
+  useEffect(() => {
+    const handleMessage = (event) => {
+      if (event.origin !== publicRuntimeConfig.legacyAppUrl) {
+        return;
+      }
+
+      if (event.data.height) {
+        setIframeHeight(event.data.height);
+      }
+    };
+    window.addEventListener('message', handleMessage);
+
+    return () => window.removeEventListener('message', handleMessage);
+  }, []);
+
+  return (
+    <Page>
+      <Page.Title>{t('search.title')}</Page.Title>
+      <Page.Content>
+        <Stack
+          height="100%"
+          width="100%"
+          css={`
+            overflow-y: auto;
+          `}
+        >
+          <Tabs<Scope>
+            activeKey={tab}
+            onSelect={handleSelectTab}
+            activeBackgroundColor={`${udbMainDarkBlue}`}
+            css={`
+              .nav {
+                margin-left: 1.5rem;
+                margin-bottom: 1.5rem;
+              }
+              .nav-item.nav-link {
+                padding: 0.4rem 1rem;
+                border: 1px solid black;
+              }
+              .nav-item {
+                margin: 0 !important;
+                border-radius: 0;
+
+                &:first-child {
+                  border-right: none;
+                  border-radius: 0.5rem 0 0 0.5rem;
+                }
+
+                &:last-child {
+                  border-radius: 0 0.5rem 0.5rem 0;
+                }
+
+                &.active {
+                  color: white;
+                }
+
+                &.active:hover {
+                  background-color: ${udbMainDarkBlue};
+                }
+              }
+            `}
+          >
+            <Tabs.Tab eventKey="event-places" title={t('search.event_places')}>
+              {tab === 'event-places' && (
+                <TabContent>
+                  <Stack flex={1}>
+                    {isClientSide && (
+                      <iframe
+                        height={
+                          iframeHeight
+                            ? `${iframeHeight}px`
+                            : `${window.innerHeight}`
+                        }
```

Comment:
I'm still a little bit on the fence on this solution, I understand the need for it since contrary to other embedded pages this one is embedded within another. But could we perhaps use Grid here or a calc to avoid having to do these recalculations back and forth with the iframe?

Reply:
In other pages, we scroll within the iframe. However, here we want to scroll the entire page so that the title and tabs do not remain sticky. I haven't found a solution to enable scrolling of the whole page without knowing the exact height of the child component. Otherwise, the scrolling is limited to within the iframe.